diff -ur original/opt/ausmt/tests/files/other.qml patched/opt/ausmt/tests/files/other.qml
--- original/opt/ausmt/tests/files/other.qml	2014-11-22 12:20:44.000000000 +0100
+++ patched/opt/ausmt/tests/files/other.qml	2014-11-22 12:21:35.000000000 +0100
@@ -6,7 +6,7 @@
     
     Image {
         anchors.centerIn: parent
-        source: "hello.png"
+        source: "hello-world.png"
     }
 }
  
diff -ur original/opt/ausmt/tests/files/simple.qml patched/opt/ausmt/tests/files/simple.qml
--- original/opt/ausmt/tests/files/simple.qml	2014-06-01 14:43:23.000000000 +0200
+++ patched/opt/ausmt/tests/files/simple.qml	2014-11-15 18:25:23.000000000 +0100
@@ -25,7 +25,7 @@
 
         PullDownMenu {
             MenuItem {
-                text: "Test !"
+                text: "Patched text !"
             }
         }
     }
